administrator/language/en-GB/plg_webservices_content.ini

```
@@ -5,3 +5,10 @@
 
 PLG_WEBSERVICES_CONTENT="Web Services - Content"
 PLG_WEBSERVICES_CONTENT_XML_DESCRIPTION="Add articles routes to the API for your website."
+PLG_WEBSERVICES_ALLOWEDVERBS_LABEL = "Allowed verbs"
```

```suggestion
PLG_WEBSERVICES_ALLOWEDVERBS_LABEL = "Allowed Verbs"
```
Labels should be capitalized.